monitoring/scd_qualifier/utils.py

```diff
@@ -0,0 +1,117 @@
+from typing import List, Union
+from geojson import Feature
+from shapely.geometry import LineString, Polygon
+from monitoring.monitorlib.typing import ImplicitDict
+import arrow
+
+class StringBasedDateTime(str):
+  """String that only allows values which describe a datetime."""
+  def __new__(cls, value):
+    if isinstance(value, str):
+      t = arrow.get(value).datetime
+    else:
+      t = value
+    str_value = str.__new__(cls, arrow.get(t).to('UTC').format('YYYY-MM-DDTHH:mm:ss.SSSSSS') + 'Z')
+    str_value.datetime = t
+    return str_value
+
+class LatLngPoint(ImplicitDict):
```

Comment:
If this is mirroring [a type in the SCD API](https://github.com/astm-utm/Protocol/blob/f6e5ee5a84cbe56d1b39187aa5ba0f67a7d8f5ec/utm.yaml#L265), let's follow [the RID style](https://github.com/interuss/dss/blob/25ccb1a6a3e732eb0d57ad87c09f3be8b71e2465/monitoring/monitorlib/rid.py#L86) and mirror the type in [`scd.py`](https://github.com/interuss/dss/blob/master/monitoring/monitorlib/scd.py).  Same comment for other types in this file mirror types defined in the API.

Reply:
I have now moved the ASTM specific definitions to `scd.py` file.